batch/batch/cloud/gcp/driver/pricing.py

```diff
@@ -17,6 +18,58 @@
 log = logging.getLogger('pricing')
 
 
+G2_CPU_SKUS = [
```

Comment:
These SKU numbers are a little opaque.  Can you change your script to include a comment with the description?

```
G2_CPU_SKUS = [
    # G2 Instance Core running in Iowa
    'D18E-3563-415F',
    ...
```